scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -16,6 +16,20 @@ namespace <%= namespace %> {
     public class NoMatchingVariantException : Exception {
         public NoMatchingVariantException(string message) : base(message) {}
     }
+
+    /// <summary>
+    /// Wraps around a shipping address and shipping identifier. Used for sending the final checkout fields in one GraphQL query.
+    /// </summary>
+    public struct ShippingFields {
+        public MailingAddressInput ShippingAddress;
```

Comment:
This does seem messy but it's due to the bigger problem of having the cart handle the state of the checkout. This struct is public because it needs to be used in `iOSNativeCheckout`